addons/website_mail_channel/static/src/snippets/s_channel/options.js

```diff
@@ -24,27 +40,26 @@ options.registry.Channel = options.Class.extend({
      *
      * @see this.selectClass for parameters
      */
-    createChannel: function (previewMode, widgetValue, params) {
-        var self = this;
+    createChannel(previewMode, widgetValue, params) {
         return wUtils.prompt({
             id: "editor_new_mail_channel_subscribe",
             window_title: _t("New Mail Channel"),
             input: _t("Name"),
-        }).then(function (result) {
+        }).then(result => {
```

Comment:
When you are making a fix in a stable version, unless you are changing the whole method, try not to change lines for conventions errors like here: you only increase the chance the forward-port does not go well. Here the whole method should be untouched.